packages/strapi-design-system/src/Icon/Icon.js

```diff
@@ -4,8 +4,8 @@ import { Box } from '../Box';
 import styled from 'styled-components';
 
 const IconWrapper = styled(Box)`
-  width: ${({ theme, width }) => theme.spaces[width]};
-  height: ${({ theme, height }) => theme.spaces[height]};
+  width: ${({ theme, width }) => theme.spaces[width] ? theme.spaces[width] : width};
+  height: ${({ theme, height }) => theme.spaces[height] ? theme.spaces[height] : height};
```

Comment:
I think you could shorten that:

```suggestion
  width: ${({ theme, width }) => theme.spaces[width] ?? width};
  height: ${({ theme, height }) => theme.spaces[height] ?? height};
```